go_modules/helpers/updater/main.go

```diff
@@ -28,7 +28,10 @@ func UpdateDependencyFile(args *Args) (interface{}, error) {
 	}
 
 	for _, dep := range args.Dependencies {
-		f.AddRequire(dep.Name, dep.Version)
+		err := f.AddRequire(dep.Name, dep.Version)
+		if err != nil {
```

Comment:
```suggestion
		if err := f.AddRequire(dep.Name, dep.Version); err != nil {
```

Nit: minimize scope of `err`.

Reply:
Yeah, I'm always torn on this one because it's less LOC, and tighter scope, but I also find it less readable. Happy to go either way.